test/services/UIBuilderSpec.scala

```diff
@@ -19,8 +19,8 @@ package services
 import base.BaseSpec
```

Comment:
Are there any tests for the handling of stacked groups?

Reply:
Currently we dont handle any stacked groups, so no handling to test